src/containers/local-spinner.js

```diff
@@ -8,13 +9,13 @@ function LocalSpinnerWrapper({ localCounter, children, showContent }) {
 }
 
 LocalSpinnerWrapper.propTypes = {
-  showContent: PropTypes.func,
-  children: PropTypes.object,
+  showContent: PropTypes.bool,
+  children: PropTypes.array,
   localCounter: PropTypes.number,
 };
 LocalSpinnerWrapper.defaultProps = {
-  showContent: () => {},
-  children: {},
+  showContent: null,
```

Comment:
showContent specifies in `propTypes` as bool. So it should have default value as `false`